jaggery-modules/rxt/module/scripts/app/app.js

```diff
@@ -980,6 +980,19 @@ var app = {};
         }
         return details.enabled ? details.enabled : false;
     };
+
+    app.isTaxonomyEnabled = function () {
+        var TaxonomyService = carbon.server.osgiService('org.wso2.carbon.governance.taxonomy.services.TaxonomyService');
+        var availability = TaxonomyService.getTaxonomyAvailability();
+        availability = String(availability);
+        availability = (availability == "true");
+        if (availability) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Currently this method not in use. removed with https://github.com/wso2/carbon-store/pull/834